test/unittest/unittest_nntrainer_models.cpp

```diff
@@ -1464,6 +1569,9 @@ INSTANTIATE_TEST_CASE_P(
 
       /**< multi output test */
       mkModelTc(multiple_output_model, "3:1:1:10", 10, ModelTestOption::COMPARE) // Todo: Enable option to ALL
+      /** resnet model */
+      // this must match training (verify only forwarding output values) for 2 iterations with tolerance 1.2e-4
+      // mkResNet18Tc(2, ModelTestOption::COMPARE) 
```

Comment:
I guess this is intended for now?

Reply:
Yes, uncomment this, and generate its corresponding golden file from input_gen. This will be resolved with #1575.